### PR TITLE
v0.2.0 and v0.1.1 release info

### PR DESCRIPTION
**Pull request by mangelajo**

### CODE REVIEW

File: src/content/releases/_index.en.md [hunk 1]
```diff
@@ -5,6 +5,29 @@ pre = "<b>4. </b>"
 weight = 15
 +++
 
+## v0.2.0 Overlapping CIDRs support
```
Comment (skitt) [inline src/content/releases/_index.en.md]:
```suggestion
## v0.2.0 Overlapping CIDR support
```

(Yes, English is weird.)

File: src/content/releases/_index.en.md [hunk 2]
```diff
@@ -5,6 +5,29 @@ pre = "<b>4. </b>"
...
+
+> This release is focused on overlapping CIDRS support between clusters
```
Comment (skitt) [inline src/content/releases/_index.en.md]:
```suggestion
> This release is focused on overlapping CIDR support between clusters
```

File: src/content/releases/_index.en.md [hunk 3]
```diff
@@ -5,6 +5,29 @@ pre = "<b>4. </b>"
...
+
+* Support for Overlapping CIDRs between clusters (globalnet)
```
Comment (skitt) [inline src/content/releases/_index.en.md]: This one however is plural, and correct.

File: src/content/releases/_index.en.md [hunk 4]
```diff
@@ -5,6 +5,29 @@ pre = "<b>4. </b>"
...
+* Enhanced e2e scripts, which will be shared between repositories in the shipyard project (ongoing work)
+* Local registry in make e2e, improves e2e deployment.
+* Refactoring for support pluggable drivers (in preparation for wireguard support)
```
Comment (skitt) [inline src/content/releases/_index.en.md]:
```suggestion
* Refactoring for support pluggable drivers (in preparation for [WireGuard](https://www.wireguard.com/) support)
```

File: src/content/releases/_index.en.md [hunk 5]
```diff
@@ -5,6 +5,29 @@ pre = "<b>4. </b>"
...
+
+
+## v0.1.1 Submariner with more light
+
+> This release has focused on stability for the Ligthouse support
```
Comment (skitt) [inline src/content/releases/_index.en.md]:
```suggestion
> This release has focused on stability for the Lighthouse support
```

File: src/content/releases/_index.en.md (hunk 1 above)
Comment (mangelajo) [reply src/content/releases/_index.en.md]: :? I trust you, but I don't get it :)

File: src/content/releases/_index.en.md [hunk 6]
```diff
@@ -5,6 +5,29 @@ pre = "<b>4. </b>"
 weight = 15
 +++
 
+## v0.2.0 Overlapping CIDR support
+
+> This release is focused on overlapping CIDR support between clusters
+
+* Support for Overlapping CIDRs between clusters (globalnet)
+* Enhanced e2e scripts, which will be shared between repositories in the shipyard project (ongoing work)
+* Local registry in make e2e, improves e2e deployment.
```
Comment (tpantelis) [inline src/content/releases/_index.en.md]:
```suggestion
* Improved e2e deployment by using a local registry.
```

File: src/content/releases/_index.en.md [hunk 7]
```diff
@@ -5,6 +5,29 @@ pre = "<b>4. </b>"
...
+* Refactoring for support pluggable drivers (in preparation for [WireGuard](https://www.wireguard.com/) support)
```
Comment (tpantelis) [inline src/content/releases/_index.en.md]:
```suggestion
* Refactoring to support pluggable drivers (in preparation for [WireGuard](https://www.wireguard.com/) support)
```

File: src/content/releases/_index.en.md [hunk 8]
```diff
@@ -5,6 +5,29 @@ pre = "<b>4. </b>"
...
+
+
+## v0.1.1 Submariner with more light
+
+> This release has focused on stability for the Lighthouse support
+
+* cleaner logging for submariner-engine
```
Comment (tpantelis) [inline src/content/releases/_index.en.md]:
```suggestion
* Cleaner logging for the submariner-engine
```

File: src/content/releases/_index.en.md [hunk 9]
```diff
@@ -5,6 +5,29 @@ pre = "<b>4. </b>"
...
+* cleaner logging for submariner-route-agent
```
Comment (tpantelis) [inline src/content/releases/_index.en.md]:
```suggestion
* Cleaner logging for the submariner-route-agent
```

File: src/content/releases/_index.en.md [hunk 10]
```diff
@@ -5,6 +5,29 @@ pre = "<b>4. </b>"
...
+* Fixed issue with wrong token stored in subm file #244
+* Added flag to disable the OpenShift CVO #235
+* Fixed several service-discovery related bugs #194 , #167
+* Fixed several panics on nil network discovery
+* Now checks if the IPs for the cluster you're trying to join overlaps with an existing one.
```
Comment (tpantelis) [inline src/content/releases/_index.en.md]:
```suggestion
* Added checks to ensure the CIDRs for joining cluster don't overlap with an existing ones.
```

File: src/content/releases/_index.en.md [hunk 11]
```diff
@@ -5,6 +5,29 @@ pre = "<b>4. </b>"
...
 
+## v0.2.0 Overlapping CIDR support
+
+> This release is focused on overlapping CIDR support between clusters
+
+* Support for Overlapping CIDRs between clusters (globalnet)
+* Enhanced e2e scripts, which will be shared between repositories in the shipyard project (ongoing work)
+* Local registry in make e2e, improves e2e deployment.
+* Refactoring for support pluggable drivers (in preparation for [WireGuard](https://www.wireguard.com/) support)
+
```
Comment (tpantelis) [inline src/content/releases/_index.en.md]: Should we mention "Added a documentation website"?

Comment (mangelajo) [reply src/content/releases/_index.en.md]: Nah, they can see if they are on the website already, it's more like a co-lateral :)

**Pull request merged**